spec/models/provider_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe Provider, type: :model do
+
+  describe "Provider" do
+    it "is provider valid" do
```

Comment:
estando bajo la descripcion del "Provider" es implicito que todos los "its" internos van a ser sobre el provider, entonces no necesitas volver a escribirlo. Al correr las pruebas las descripciones se encadenan jerarquicamente. Es decir esta descripcion se uniria y diria:     "Provider is provider valid". Como que es un poco redundante una oracion que diga "El proveedor es proveedor valido". seria como: "is valid" para que al final la oracion diga "Provider is valid"